model/fv_dynamics.F90

```diff
@@ -1097,16 +1198,24 @@ end subroutine Rayleigh_fast
 
 
 
- subroutine Rayleigh_Super(dt, npx, npy, npz, ks, pm, phis, tau, u, v, w, pt,  &
+ subroutine Rayleigh_Super(dt, npx, npy, npz, ks, pm, phis, tau, tau_w, u, v, w, pt,  &
+#ifdef MULTI_GASES
+                           q, ncnst,  &
+#endif
```

Comment:
Why insert this in the middle of the argument list and not at the end?

Reply:
This is not changed. There are many similar 

> Also, there are a few source files where you are adding in whitespace at the end of a line. Could you please make sure to remove all unnecessary whitespace addition. In fv_control.F90 for example, I now see a difference on the lines defining consv_te and tau which is related to extra whitespace at the end of the line.



> You are computing and passing time_total to fv_dynamics. It would be easier to use the fv_time variable from fv_diagnostics_mod and the get_time function to compute the time_total value where needed. There is a plan to utilize the existing Atm%Time variable for this purpose in the future.

time_total is calculated in fv_dynamics now.

Reply:
This part is not changed since similar #ifdef was used in many other places. I guess it is to avoid the #else part.